sys/linux/auto.txt

```diff
@@ -6495,12 +8598,68 @@ udp_timeout_nla_policy$auto_nf_conntrack_proto_udp [
 	CTA_TIMEOUT_UDP_REPLIED		nlattr[CTA_TIMEOUT_UDP_REPLIED, int32]
 ] [varlen]
 
+unewset$auto_record {
+	sig	array[int32, 1]
+}
+
+uprobe_multi$auto_record {
+	path		intptr
```

Comment:
This is declared as `__aligned_u64`. Can we infer that this should be int64?

Reply:
Done!